save-plugins/fix-plugin/src/commonMain/kotlin/org/cqfn/save/plugins/fix/FixPlugin.kt

```diff
@@ -48,38 +48,50 @@ class FixPlugin(
 
         val fixPluginConfig = testConfig.pluginConfigs.filterIsInstance<FixPluginConfig>().single()
         val generalConfig = testConfig.pluginConfigs.filterIsInstance<GeneralConfig>().singleOrNull()
-        return files
-            .map { it.first() to it.last() }
-            .map { (expected, test) ->
-                val testCopy = createTestFile(test)
-                val execCmd = "${(generalConfig!!.execCmd)} ${fixPluginConfig.execFlags} $testCopy"
-                val executionResult = try {
-                    pb.exec(execCmd, null)
-                } catch (ex: ProcessExecutionException) {
-                    return@map TestResult(
-                        listOf(expected, test),
-                        Fail("${ex::class.simpleName}: ${ex.message}", ex::class.simpleName ?: "Unknown exception"),
-                        DebugInfo(null, ex.message, null)
-                    )
-                }
-                val stdout = executionResult.stdout.joinToString("\n")
-                val stderr = executionResult.stderr.joinToString("\n")
 
+        return files.chunked(fixPluginConfig.batchSize ?: 1).map { chunk ->
```

Comment:
```suggestion
        return files.chunked(fixPluginConfig.batchSize!!).map { chunk ->
```
Since batchSize already has a default value from `validateAndSetDefaults` method, there is no need to repeat it here. We can assume that the contract is, that config will have all required fields properly set by this time

Reply:
Fixed